menu/menu_setting.c

```diff
@@ -1300,8 +1300,10 @@ static int setting_uint_action_left_crt_switch_resolution_super(
       case 0:
          *setting->value.target.unsigned_integer = 3840;
          break;
-      case 1920:
+      case 1: /* for dynamic super resolution switching - best fit */
```

Comment:
This falls-through. I'm not sure if this is what you intended but it looks wrong at least. You need to probably insert a break; here.